resources/guides/contributing.md

```diff
@@ -4,3 +4,113 @@ title: How To Contribute
+If the article you're updating contains a "Last Updated" date at the start of it, make sure to update it with the current date! Make sure to also follow the [style guide](/resources/guides/style) when updating an article.
+
+## Creating New Articles
+
+Creating new articles is a bit more complex than updating existing ones, but if you follow the pattern of other articles, you'll catch on quickly.
+
+The first thing you need to consider is where your article should be located on the wiki, and where it should be located in the actual code. Luckily, the layout of files and folders in the wiki codebase usually follows the layout of the site itself. So, if you wanted to create a new `guide` article, you'd create a new `.md` file under `resources/guides`.
+
+When opening up any of the existing markdown files, you'll probably notice that each one starts off with a section that looks like this:
+
+```md
+---
+id: contributing
+title: How To Contribute
+sidebar_label: How To Contribute
+slug: /guides/contributing
+---
+```
+
+Including this top section before beginning your article is vital if you want it to show up on the wiki. Each field is fairly self explanatory, but here's a quick rundown anyways:
+
+-   **id**: This is a descriptive string that uniquely identifies your article. It's useful for sidebars.
+-   **title**: The title of your article, this is automatically rendered as an `h1` at the beginning of your article.
+-   **sidebar_label**: Self explanatory, shows up on the sidebar to link to your article.
+-   **slug**: This refers to what is appended to the url when going to your article. Sometimes this can be confusing to work with so it's best to follow the pattern of other articles in the same directory. For example, if you're creating a guide about "Topic A", you might make the slug `/guides/topic-a`.
+
+Once you have this section completed, you now need to edit the relevant `XXXX.sidebars.js` file to create a link for it in the sidebar of whatever section you're writing the article for. For example, if you're creating a new guide you would update `resources_sidebars.js`. Look for items near where you'd like to place your article:
+
+```js
+items: [
+    ...
+    "guides/style_guide",
+    "guides/contributing",
+    ...
+],
+```
+
+Here, you can see that to add a new guide, you'll need to add an item here. If your article has an id of "someID", you'll add the following:
+
+```js
+"guides/someID",
+```
+
+If, say, you wanted to add an article about CSS, you'll notice the sidebars file is a bit different:
+
+```js
+    indexNav: [
+        {
+            type: "doc",
+            id: "about",
+        },
+        {
+            type: "doc",
+            id: "meetTheBoard",
+        },
+    ],
+```
+
+Here, to add a new article, say with id of "meetRyan", you would add the following to the list:
+
+```js
+{
+    type: "doc",
+    id: "meetRyan",
+},
+```
+
```

Comment:
I think we should also tell them to update the index page with the subpage map just to keep that updated as well

Reply:
good idea, I'll add that